index.html

```diff
@@ -46,7 +46,7 @@
           <li><a class="nav-link scrollto" href="#venue">Venue</a></li>
           <li><a class="nav-link scrollto" href="#hotels">Hotels</a></li>
           <li><a class="nav-link scrollto" href="#supporters">Sponsors</a></li>
-          <li><a class="nav-link scrollto" href="/code-of-conduct.html">Code of Conduct</a></li>
+          <li><a class="nav-link scrollto" href="code-of-conduct.html">Code of Conduct</a></li>
```

Comment:
@basiima what is the essence of this change? why are we changing from `href="/code-of-conduct.html"` to `href="#"`